### PR TITLE
Corrected support for max. resolution

### PR DESCRIPTION
**Pull request by ktunkiewicz**

### CODE REVIEW

File: dist/jpeg_camera.js [hunk 1]
```diff
@@ -230,6 +230,8 @@ var JpegCameraBase = function () {
     //   message will be passed as the first argument. Inside the callback camera
```
Comment (e-oloughlin) [inline dist/jpeg_camera.js]: This is my first time reviewing something from this repo, but do we check in dist files? Are they built ?

File: package-lock.json [hunk 2]
```diff
@@ -1,14 +1,22 @@
 {
   "name": "jpeg-camera-es6",
-  "version": "1.0.7",
+  "version": "1.0.11",
   "lockfileVersion": 1,
   "requires": true,
   "dependencies": {
+    "accepts": {
+      "version": "1.3.5",
+      "resolved": "http://nexus-test.currencyfair.com/nexus/repository/npm-all/accepts/-/accepts-1.3.5.tgz",
```
Comment (niallmccrudden) [inline package-lock.json]: might want to update to use nexus?

**Comment by niallmccrudden** [comment]
👍

File: package-lock.json (hunk 2 above)
Comment (ktunkiewicz) [reply package-lock.json]: ah f**, how did that get here... that's a public repo... I'll fix that asap

File: dist/jpeg_camera.js (hunk 1 above)
Comment (ktunkiewicz) [reply dist/jpeg_camera.js]: These are build using webpack. Of course you can check them for some obvious problems, but basically this is autogenerated.

**Comment by e-oloughlin** [comment]
👍

**Pull request merged**